pysat/instruments/icon_mighti.py

```diff
@@ -53,8 +53,8 @@
 platform = 'icon'
 name = 'mighti'
 tags = {'level_2': 'Level 2 public geophysical data'}
-sat_ids = {'green': ['level_2 Green Line'],
-           'red': ['Level_2 Red Line']}
+sat_ids = {'green': ['level_2'],
+           'red': ['level_2']}
```

Comment:
Should this be in its own PR?  It's technically a breaking change for anyone who has build code around this instrument.

Reply:
Since this data is not publicly available, I think it's fine here.  The whole routines will have to be rewritten once the data is released anyway, since the file names will be different from what is here.  This is being changed in this PR as the sat_ids are improperly set up and fail the tests otherwise.

As I understand it, the code being used by team members is not the same as that on github.  @rstoneback?

Reply:
I started updating some things for EUV, FUV, and MIGHTI centered around downloads. There was an aspect of MIGHTI loading I didn't like which I intended to fix. The effect is I haven't sent that code out to anybody yet. The public versions are effectively zombies. They were compliant when written but the standards have changed since then.

The intention for public ICON support is for level-2 files even if the code didn't properly reflect that.